solidity/contracts/bridge/Wallets.sol

```diff
@@ -362,19 +372,12 @@ library Wallets {
         bytes20 walletPubKeyHash
     ) internal {
         Wallet storage wallet = self.registeredWallets[walletPubKeyHash];
+        // Initialize the moving funds process.
+        wallet.state = WalletState.MovingFunds;
+        /* solhint-disable-next-line not-rely-on-time */
+        wallet.movingFundsRequestedAt = uint32(block.timestamp);
 
-        if (wallet.mainUtxoHash == bytes32(0)) {
-            // If the wallet has no main UTXO, that means its BTC balance
-            // is zero and it should be closed immediately.
-            closeWallet(self, walletPubKeyHash);
```

Comment:
If we decide never close the wallet in the `moveFunds` function then we will have to update the description of `notifyRedemptionTimeout` as it states the wallet can be closed if it no longer has the main UTXO.

Reply:
No longer relevant because of: https://github.com/keep-network/tbtc-v2/commit/a704bf38a45f7ba6f64ed597a13603b40e6e1c08